examples/shapes_and_collections/plot_path_patch.py

```diff
@@ -2,6 +2,9 @@
 ================
 PathPatch object
 ================
+
+This example shows how to create Paths and PathPatch objects through
```

Comment:
backticks might work automatically for cross references. There's unfortunately no easy way to check this just yet, but we'll see once I merge this branch into the sphinx-gallery branch.